### PR TITLE
HIVE-25637: Hive on Tez: inserting data failing into the non native hive external table managed by kafka storage handler

### PR DESCRIPTION
**Pull request by abstractdog**
### What changes were proposed in this pull request?
Supporting INSERT into a kafka table in a secure cluster. Further details are in JIRA description and in [this comment](https://issues.apache.org/jira/browse/HIVE-25637?focusedCommentId=17434895&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-17434895)

### Why are the changes needed?
INSERT into a kafka table failed.

### Does this PR introduce _any_ user-facing change?
No.

### How was this patch tested?
Tested on cluster.

### CODE REVIEW

File: ql/src/java/org/apache/hadoop/hive/ql/exec/tez/DagUtils.java [hunk 1]
```diff
@@ -311,10 +325,18 @@ private void getKafkaCredentials(MapWork work, DAG dag, JobConf conf) {
         break;
       }
     }
+
+    for (TableDesc tableDesc : fileSinkTableDescs) {
+      kafkaBrokers = (String) tableDesc.getProperties().get("kafka.bootstrap.servers"); //FIXME: KafkaTableProperties
+      if (kafkaBrokers != null && !kafkaBrokers.isEmpty()) {
```
Comment (maheshk114) [inline ql/src/java/org/apache/hadoop/hive/ql/exec/tez/DagUtils.java:331]: The common code can be reused instead of another copy

**Review by maheshk114 — APPROVED** [review]

**Review by maheshk114 — CHANGES_REQUESTED** [review]
Please fix the review comment

**Review by maheshk114 — APPROVED** [review]

**Comment by abstractdog** [comment]
TestReplWithJsonMessageFormat failure is unrelated

**Comment by abstractdog** [comment]
merged to master, thanks @maheshk114  for the review!

**Pull request merged**